app/src/main/java/org/openobservatory/ooniprobe/utils/AlarmReceiver.java

```diff
@@ -16,9 +16,9 @@ public void onReceive(Context context, Intent intent) {
         //System.out.println(DEBUG_TAG);
         NotificationHandler.sendNotification(context, context.getString(R.string.local_notifications_text));
         TestData.getInstance(context, null);
-        TestData.doNetworkMeasurements(context, "web_connectivity");
-        TestData.doNetworkMeasurements(context, "http_invalid_request_line");
-        TestData.doNetworkMeasurements(context, "ndt_test");
+        TestData.doNetworkMeasurements(context, "web_connectivity", null);
+        TestData.doNetworkMeasurements(context, "http_invalid_request_line", null);
+        TestData.doNetworkMeasurements(context, "ndt_test", null);
```

Comment:
@hellais do we want to run more tests here?

Reply:
Good question

Reply:
Ping @hellais

Reply:
We want to run at minimum "http_header_field_manipulation" too. Maybe also "dash". I would say that as it is now, we should probably have a list of all the supported tests somewhere (it probably already exists) and just run them all.